consensus/validator.go

```diff
@@ -273,6 +278,21 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 	consensus.aggregatedCommitSig = aggSig
 	consensus.commitBitmap = mask
 
+	// If we already have a committed signature received before, check whether the new one
```

Comment:
I am a little confused about these logic:

1. Is it true that leader can possibly send out multiple committed message? And why multiple committed is required for pipelining?
2. If the user shutdown the node before receiving the full commit sig, is it true that it could possibly cause database corruption in `commitSig`?

Reply:
1. yea, leader will send out the 67% committed message first. And later could possibly also send out the 100% committed if the next consensus doesn't happen in time. Because it better pass the 100% committed sig to the next leader.

2. No, the db operation is atomic, it's just updating the commitSig for the block. Nothing else. The original logic of updating the block with the commit sig atomically is still there. This is just to potentially update 67% committed with 100% committed sigs

Reply:
I see. Thanks for clarification.